scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -410,14 +410,20 @@
   [^TaskScheduler fenzo considerable offers db fid]
   (log/debug "Matching" (count offers) "offers to" (count considerable) "jobs with fenzo")
   (let [t (System/currentTimeMillis)
+        _ (log/debug "offer to scheduleOnce" offers)
```

Comment:
just for style, can't we just move these log statements above the let block?  Fewer anonymous let variables